routes/api/collectors.js

```diff
@@ -0,0 +1,24 @@
+let express = require('express');
+let router = express.Router();
+let models = require('../../models/index');
+
+router.post('/', function(req, res, next) {
+  models.savePayload(req.body, (collectorKey) => {
+        res.json({
```

Comment:
I wanted to make a serializer but noticed that I do not have a model and that the only data available to respond to the client with is the id and the kind attributes.